zygarde.js

```diff
@@ -53,17 +56,13 @@ for (const {
 }
 
 const client = new discord.Client({ disableEveryone: true });
+
+// subscribe to all zephyr classes, both the main bridged class and
+// its related class, via the tripplet <c,*,*>
 zephyr.subscribe(
-  [].concat.apply(
-    [],
-    settings.classes.map(({ zephyrClass, zephyrRelatedClasses = {} }) =>
-      [zephyrClass, ...Object.keys(zephyrRelatedClasses)].map(c => [
-        c,
-        "*",
-        "*"
-      ])
-    )
-  ),
+  [].concat.apply([], settings.classes.map(({ zephyrClass, zephyrRelatedClasses = {} }) =>
+    [zephyrClass, ...Object.keys(zephyrRelatedClasses)].map(c => [c, "*", "*" ])
+  )),
```

Comment:
Prettier is going to want to snap this back to how it was; I was only running it manually, but still. To prevent that, try adding a line comment so it chooses better formatting:
```js
zephyr.subscribe(
  [].concat.apply(
    [],
    settings.classes.map(({ zephyrClass, zephyrRelatedClasses = {} }) =>
      [zephyrClass, ...Object.keys(zephyrRelatedClasses)]
        // Make each into a zephyr triplet
        .map(c => [c, "*", "*"])
    )
  )
);
```